docs/dictionary/function/iphoneFileDataProtection.lcdoc

```diff
@@ -24,10 +24,11 @@ A string containing the path to the file.
 
 Returns (enum):
 
--   none: 
--   complete: 
--   complete unless open: (iOS 5 and later only)
--   complete until first user authentication: (iOS 5 and later only)
+-   empty: The file was not accessible or argument not specified.
+-   none: No protection.
+-   complete: The file is not accessible, for read or write, while the device is locked.
+-   complete unless open: The file is fully protected when the device is locked, unless it was already open.
+-   complete until first user authentication: The file is fully protected until the user unlocks the device for this first time.
 
```

Comment:
These will also have to be unwrapped because of bug 18657. Also, the values (other than `empty`) need to be in double-quotes.